doc/tutorials/platformer/app/lib/ember_quest.dart

```diff
@@ -57,19 +57,19 @@ class EmberQuestGame extends FlameGame
   void loadGameSegments(int segmentIndex, double xPositionOffset) {
     for (final block in segments[segmentIndex]) {
       final component = switch (block.blockType) {
-        GroundBlock => GroundBlock(
+        GroundBlock _ => GroundBlock(
```

Comment:
I think there might be a better way of doing type checks in switches

Reply:
the other option suggest is this:
```
        const (GroundBlock) => GroundBlock(
            gridPosition: block.gridPosition,
            xOffset: xPositionOffset,
          ),
```
I thought the underscore looked cleaner as it implies in my mind a pattern matching variable that is just not captured